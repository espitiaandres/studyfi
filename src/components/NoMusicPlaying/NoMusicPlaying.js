//
//  NoMusicPlaying.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import './NoMusicPlaying.css';

const NoMusicPlaying = () => {
    return (
        <div className="noMusicPlayingMessage">
            <h1>hmm...</h1>
            <p>
                It seems that you're on Spotify but you're not currently listening to music. Listen to music for something to appear here!
            </p>
        </div>
    )
}

export default NoMusicPlaying;
