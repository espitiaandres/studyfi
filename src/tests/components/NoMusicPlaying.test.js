//
//  NoMusicPlaying.test.js
//  react-spotify-player
//
//  Created by Andres Espitia.
//  Copyright Â© 2020 Andres Espitia. All rights reserved.
//

import React from 'react';
import { shallow } from 'enzyme';
import NoMusicPlaying from '../../components/NoMusicPlaying/NoMusicPlaying';

test('should render NoMusicPlaying component correctly', () => {
    const wrapper = shallow(<NoMusicPlaying />);
    expect(wrapper).toMatchSnapshot();
});
