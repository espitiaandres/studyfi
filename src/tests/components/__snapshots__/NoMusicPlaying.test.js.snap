// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoMusicPlaying component correctly 1`] = `ShallowWrapper {}`;
